crates/relayer/src/chain/axon.rs

```diff
@@ -955,33 +973,28 @@ impl AxonChain {
         let receipts: Receipts = tx_receipts.into();
         let receipt_proof = receipts.generate_proof(receipt.transaction_index.as_usize());
 
-        let (block, state_root, proof, mut validators) = self
+        let (block, state_root, block_proof, mut validators) = self
             .rt
             .block_on(self.get_proofs_ingredients(block_number))?;
 
         // check the validation of receipts mpt proof
         let key = rlp::encode(&receipt.transaction_index.as_u64());
-        let verify_mpt = trie::verify_proof(
-            &receipt_proof,
-            block.header.receipts_root.as_bytes(),
-            &key,
-            &receipt.rlp_bytes(),
-        );
-        if !verify_mpt {
-            return Err(Error::rpc_response("unverified receipts mpt".to_owned()));
-        }
+        // axon_tools::verify_trie_proof(block.header.receipts_root, &key, receipt_proof.clone())
```

Comment:
What the purpose of this comment?

Reply:
because of the Axon's bug, the commented code will fail until the bug fixed

Reply:
Can you leave a comment on this? `#FIXME skip Axon trie verify for now`